utils/DeployedContractsHelpers.ts

```diff
@@ -222,6 +224,10 @@ export function getBDStableFiat(symbol: string): string {
   return bdStablesContractsDetails[symbol].fiat;
 }
 
+export function getBDStableChainlinkPriceFeed(symbol: string): string | undefined {
```

Comment:
Why also support undefined?

Reply:
for USD

Reply:
we don't need price feed for BDUS as the price is 1:1